src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/65663691/d94bdaa1-f600-4149-b96d-5e3da976e39a)
but then i will have that:

Reply:
see the difference between `static` and `non-static` methods and how to utilize them. 

It is because you need to instantiate the object previously.

Reply:
everything worked out for me! thanx